docs/operational_guide/etcd.md

```diff
@@ -0,0 +1,81 @@
+# etcd
```

Comment:
Do we want to have recs on how many external etcd hosts one should have? is there some sort of formula based on the size of the m3db cluster?

Reply:
There's some etcd docs on the subject we can link to: https://github.com/etcd-io/etcd/blob/v3.3.11/Documentation/faq.md#what-is-failure-tolerance